cypress/e2e/models/migration/dynamic-report/issues/issues.ts

```diff
@@ -107,4 +108,52 @@ export class Issues {
             click(selector);
         }
     }
+
+    public static unfold(name: string) {
+        cy.contains(tdTag, name)
+            .closest(trTag)
+            .within(() => {
+                cy.get("[id^=expandable]").then(($button) => {
+                    if (!$button.hasClass("pf-m-expanded")) {
+                        $button.trigger("click");
+                    }
+                });
+            });
+    }
+
+    public static validateAllFields(issue: AppIssue): void {
+        const sections = {
+            totalAffectedApps: "Total affected",
+            targetTechnologies: "Target technologies",
+            sourceTechnologies: "Source technologies",
+            ruleSet: "Rule set",
+            rule: /^Rule$/,
+            labels: "Labels",
+        };
+
+        Issues.unfold(issue.name);
+        Issues.validateSection(sections.totalAffectedApps, button, /\d - View affected /);
+        Issues.validateSection(sections.targetTechnologies, span, issue.targets);
+        Issues.validateSection(sections.sourceTechnologies, div, issue.source);
+        Issues.validateSection(sections.ruleSet, div, issue.ruleSet);
+        // @ts-ignore
```

Comment:
This annotation can be safely removed.

Reply:
I don't wanna remove it, otherwise Webstorm is arguing about a mistake there. But there is NO mistake and all just works, so I wanted to suppress this.

Reply:
Now that you mention it, there is an error, you're trying to pass an instance of Regexp to a function that ony accepts string in that parameter. Even if no error is displayed in runtime its an actual error.

Just add the option to receive also a RegExp in that parameter as you already did with the other one

```typescript
 private static validateSection(
        title: string | RegExp,
        contentSelector: string,
        content: string | string[] | RegExp
 ): void {
```

Reply:
Fixed, thanks for an advise.